test/server/python_deps.jl

```diff
@@ -0,0 +1,32 @@
+# this script uses the default Conda settings to get python, pyopenssl, and pyftpdlib
+# if you already have an acceptable version of python and pip installed it will attempt to use those instead
+
+using Conda
+using Compat
+
+py_version(python::AbstractString) = chomp(readstring(`$python -c "import platform; print(platform.python_version())"`))
```

Comment:
It isolates the version number instead of having to use a regex to grab it after the python call. I'll probably end up changing it later.
The result from this is "2.7.11" instead of "Python 2.7.11" on my computer.